crux-http-server/src/crux/http_server.clj

```diff
@@ -563,19 +611,30 @@
                                                                    (instant/read-instant-date))})
               results (api/q db query)]
           {:status 200
-           :body (if html?
-                   (let [links (link-top-level-entities db  "/_entity" results)
-                         {:keys [limit offset]} query
-                         prev-offset (when-not (zero? offset)
-                                       (max 0 (- offset limit)))
-                         next-offset (when (= limit (count results))
-                                       (+ offset limit))
-                         prev-next-page (resolve-prev-next-offset query-params prev-offset next-offset)]
-                     (html5 (query->html links query results prev-next-page)))
-                   results)})
+           :body (cond
+                   html? (let [links (link-top-level-entities db  "/_entity" results)
+                               {:keys [limit offset]} query
+                               prev-offset (when-not (zero? offset)
+                                             (max 0 (- offset limit)))
+                               next-offset (when (= limit (count results))
+                                             (+ offset limit))
+                               prev-next-page (resolve-prev-next-offset query-params prev-offset next-offset)]
+                           (raw-html
+                            {:body (query->html links query results prev-next-page)
+                             :title "/_query"
+                             :options options}))
+                   (get query-params "link-entities?") {"linked-entities" (link-top-level-entities db  "/_entity" results)
```

Comment:
we won't be able to do this once we're streaming queries - this'll bring all of the results into memory. I wonder whether there's a better way to include them inline, as you iterate through the query results...? :thinking: